.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Bug report
 about: A report of an error in a recent systemd version
-
+SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
hmm, this is weird. this is a template for to be created github issues. can we do this? add a license to everyone's issue submissions?

Reply:
This is metadata about the template itself, no? If you click on "new issue" you don't see these, they are not copied over.

Reply:
ie: click here https://github.com/systemd/systemd/issues/new?assignees=&labels=&template=Bug_report.md and see that there's no "name" or "about", it's just metadata fields